### ISSUE TITLE
Edit date parameters manually

### ISSUE THREAD

**Issue by jaredreich** [jaredreich/notie#115]
e.g. Month as a select, day/year as inputs.

Should work in parallel with up/down buttons.

**Comment by RobThree** [comment]
Since you locked #114 but I would still like to reply:

> It's your wording buddy, "DateTime Issues", "counter-intuitive", "a pain", "if that's not an option then don't return a time-part".

> "DateTime Issues"

There _are_ some datetime issues (you seem to agree since you opened #115). I also don't see how you can take offence on that?

> "counter-intuitive"

Tell me it isn't?

> "a pain"

Tell me it isn't? Should I have worded it "very unpleasant"? Would that be better?

> "if that's not an option then don't return a time-part"

"if that's not an option then **maybe consider** don't return a time-part"; my bad. That's what I meant but didn't type.

> This project doesn't owe you anything, show some respect if you want respect in return.

I never said you / the projects owes me anything? I merely reported some findings? I don't see what you're taking offence on? 

> "Consider one person's strong opinions, then discard them as out of scope" is more like it.

Sure thing; then just discard them. Don't argue about it. I just gave you my $0.02 in the initial issue (#114). I _really_ don't see what I did to you?

> All of this aside, I do appreciate you taking the time to file an issue.

You're welcome.

> You could however work on your people skills my friend!

\*mumbles something about a pot and kettle, again...\*

But I'll leave it at this. Let's just agree to disagree and leave it at that.

**Comment by jaredreich** [comment]
My goodness, you are exhausting!

**Comment by jaredreich** [comment]
Implemented in https://github.com/jaredreich/notie/releases/tag/v4.2.0
